### ISSUE TITLE
Cannot connect to host finviz.com?

### ISSUE THREAD

**Issue by windowshopr** [mariostoev/finviz#27]
Wondering if anyone else has had this problem lately or if it's something I'm missing?

```
raise client_error(req.connection_key, exc) from exc
aiohttp.client_exceptions.ClientConnectorError: Cannot connect to host finviz.com:443 ssl:None [Connect call failed ('IP_ADDRESS_HERE', 443)]
```

The error happens when the Scraper() function is used. Thoughts?

**Comment by windowshopr** [comment]
Scratch that, might have been an issue on my end. It started working randomly. Thanks.

**Comment by windowshopr** [comment]
@mariostoev  I noticed your help with the last issue so figured I'd tag you in this one. The API is working for me just wonderfully, but I noticed that the API returns the above error message, if your search results in a big number of stocks (like more than 500 for sure, I'm not sure exactly what the cut off is yet). So if you include some filters that results in a screen of say 2,000 stocks, I get the above error message. Is this a part of the API or is there a cut off we should be mindful of? If I run the SP500 filter, it will download all of them. Thanks in advance!
